script/1_0_2current_en_script/Ciel/Day 7/07_01_CIEL07_1

```diff
@@ -0,0 +1,110 @@
+<Page1>
+―――For some reason, waking up this morning felt good.
+<Page2>
+I’m fully conscious from the moment I open my eyes.
+My hands, which usually don’t work for a few seconds after I wake up no matter how much effort I put into it, went to the glasses beside my pillow without pause. With them on, my vision becomes normal.
+<Page3>
+"...Well, what a surprise. I’m feeling incredibly well this morning...?"
+<Page4>
+Is it because I slept my way through yesterday? Or did Professor Arach’s IV drip work?
+Whatever it was, I feel really good. Too good in fact, that it’s making me a bit anxious.
+<Page5>
+There’s a knock on the door.
+After around ten seconds, Hisui entered the room.
+<Page6>
+"Pardon the intrusion."
+"Morning, Hisui. As always, thanks."
+<Page7>
+"Good morning, Shiki-sama. I see that you are already awake."
+"I slept like a log yesterday, so yeah. I’ll head down to the dining hall after I get changed. I’m feeling really hungry, so could you let Kohaku-san know to make my breakfast portions a bit bigger than usual?"
+<Page8>
+"I understand. I’m sure Nee-san will be delighted."
+Hisui bowed and exited the room.
+The way she carried herself felt a bit different than usual, to the point that I accidentally called out to her midway.
+<Page9>
+"Hisui, did something good happen?"
+"Yes. Your complexion has improved significantly, Shiki-sama. As your personal caretaker, there is no greater joy in this world than seeing you well."
+<Page10>
+So she said, and promptly exited the room.
+"――――――"
+Perhaps, to Hisui, those words were to be spoken as a natural part of her occupation. However, it felt like a sneak attack to me.
+"Ah, um... Yeah, well, thanks."
+As I scratched my flushing cheeks, I blabbered out something only I could hear.
+<Page11>
+"―――No, now’s not the time for this.
+I’m feeling good in a good while, I need to change quickly."
+<Page12>
+I switched my gears and shifted to preparing for my morning.
```

Comment:
Good use of the phrase but drop "shifted". Tense also  needs some love:
> 気を取り直して朝の支度をする。
I switch gears and start preparing for the morning.